### DISCUSSION TITLE
Web3auth custom auth verifier creation problem.

### DISCUSSION THREAD

**Discussion by smartdev0530** [Web3Auth/Web3Auth#912] [Help]
I've created my custom auth verifier, and web3auth says I will get email notification within 20 min when processing is done.
i've waited more than 2 hours, but not reply yet
![image](https://user-images.githubusercontent.com/91709558/203104437-db3d0e68-8551-429f-bf50-50bc3d715c32.png)

**Comment by Saviour1001** [comment]
Can you try deleting this verifier and creating one again? 
The feature is working fine.

**Comment by tanguyenvn** [comment]
Hi, the reason of taking more than 20 minutes is that during that time, we were migrating from Ropsten to Goerli for our internal smart contracts of Testnet. I've just checked and found the verifier were Live since 22 November 5:31:10 UTC.
